src/hotspot/share/runtime/lightweightSynchronizer.cpp

```diff
@@ -771,17 +766,16 @@ void LightweightSynchronizer::exit(oop object, BasicLock* lock, JavaThread* curr
 }
 
 // LightweightSynchronizer::inflate_locked_or_imse is used to to get an inflated
-// ObjectMonitor* with LM_LIGHTWEIGHT. It is used from contexts which require
-// an inflated ObjectMonitor* for a monitor, and expects to throw a
-// java.lang.IllegalMonitorStateException if it is not held by the current
-// thread. Such as notify/wait and jni_exit. LM_LIGHTWEIGHT keeps it invariant
+// ObjectMonitor* when lightweight locking is used. It is used from contexts
```

Comment:
I guess you don't need the phrase "when lightweight locking is used".

Reply:
Even calling it "lightweight locking" is no longer needed.

Reply:
I think the name "lightweight locking" is used for the file name and class name, so the name is okay.  It does help if you're trying to understand the history of the locking algorithm.